test/images/pets/peer-finder/Makefile

```diff
@@ -12,16 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: push
+SRCS = peer-finder
+ARCH ?= amd64
+TARGET ?= $(CURDIR)
+GOLANG_VERSION ?= latest
+SRC_DIR = pets/$(notdir $(shell pwd))
```

Comment:
why is this one different from the other Makefiles (which just do `pets/peer-finder`)?